Big-Spender/readme.md

```diff
@@ -93,6 +97,7 @@ CREATE YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERY HERE
+SELECT SUM(amount) FROM spends WHERE date between '2021-03-31' and '2021-05-01
```

Comment:
we need to know by each supplier? How would this work? You might need to do a join here ;)